assets/data/social.yml

```diff
@@ -495,15 +497,39 @@ bilibili:
 
 # 063: Email
 email:
-  Weight: 63
+  Weight: -4
   Template: mailto:%v
   Title: Email
   Icon:
-    Class: far fa-envelope fa-fw
+    Class: fa fa-envelope fa-fw
+
+# 064: Phone
+phone:
+  Weight: -3
+  Template: tel:%v
+  Title: Phone
+  Icon:
+    Class: fa fa-phone fa-fw
+
+# 065: PGP
+pgp:
+  Weight: -2
+  Prefix: https://pgp.key-server.io/
```

Comment:
`Rel: pgpkey` would be very welcome here:
https://indieauth.com/pgp

Reply:
I added, but i cant see any effect on the site. `rel=` was not modified.
May i did something wrong?

Reply:
Oh, I thought I merged that one upstream, but turns out I didn't. :)

Reply:
For this to work, a change to `layouts/partials/plugin/social.html` is required:

instead of
```
{{- dict "Destination" $destination "Rel" "me" | merge . | partial "plugin/link.html" -}}
```

needs to be:
```
{{- $rel := "me" -}}
{{- with .Rel -}}{{- $rel = printf "%s %s" . $rel -}}{{- end -}}
{{- dict "Destination" $destination "Rel" $rel | merge . | partial "plugin/link.html" -}}
```